lib/include/openturns/OTishigamiusecase.hxx

```diff
@@ -0,0 +1,176 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief The header file that defines the Ishigami Use Case
+ *
+ *  Copyright 2005-2024 Airbus-EDF-IMACS-ONERA-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#ifndef OPENTURNS_OTISHIGAMIUSECASE_HXX
+#define OPENTURNS_OTISHIGAMIUSECASE_HXX
+
+#define _USE_MATH_DEFINES
+#include <cmath>          // For M_PI
+
+#include "openturns/OTconfig.hxx"
+#include "openturns/Distribution.hxx"
+#include "openturns/SymbolicFunction.hxx"
+#include "openturns/Function.hxx"
+
+BEGIN_NAMESPACE_OPENTURNS
+
+class IshigamiUseCase
+{
+  public:
+    /* Default constructor */
+    IshigamiUseCase() : dimension_(3), a_(7.0), b_(0.1)
+    {
+      // Create the Ishigami function
+      Description inputVariables(dimension_);
+      inputVariables[0] = "xi1";
+      inputVariables[1] = "xi2";
+      inputVariables[2] = "xi3";
+      Description formula(1);
+      formula[0] = (OSS() << "sin(xi1) + (" << a_ << ") * (sin(xi2)) ^ 2 + (" << b_ << ") * xi3^4 * sin(xi1)");
+      model_ = SymbolicFunction(inputVariables, formula);
+  
+      // Create the input distribution
+      Collection<Distribution> marginals(dimension_);
+      marginals[0] = Uniform(-M_PI, M_PI);
+      marginals[1] = Uniform(-M_PI, M_PI);
+      marginals[2] = Uniform(-M_PI, M_PI);
```

Comment:
use the simplified ctor since elements are identical

Reply:
You mean
```
Collection<Distribution> marginals(dimension_, Uniform(-M_PI, M_PI));
```

Reply:
Ok. Fixed at 4578d84604266a4e6d4eb217f0aec3dfd83bc6bb